chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -370,10 +349,10 @@ public static void exportToZip(File sceneDirectory, String name, File targetFile
    * scene and after scene canvas size changes.
    */
   public synchronized void initBuffers() {
-    frontBuffer = new BitmapImage(width, height);
-    backBuffer = new BitmapImage(width, height);
-    alphaChannel = new byte[width * height];
-    samples = new double[width * height * 3];
+    frontBuffer = new BitmapImage(canvasConfig.getWidth(), canvasConfig.getHeight());
+    backBuffer = new BitmapImage(canvasConfig.getWidth(), canvasConfig.getHeight());
+    alphaChannel = new byte[canvasConfig.getPixelCount()];
+    samples = new double[canvasConfig.getPixelCount() * 3];
```

Comment:
Maybe we could add a `Canvas` class that has these buffers and then one could apply the config to it?
`CanvasConfig` currently is the only class where we split config and logic, eg. sky bundles both. Not sure what's the better approach, but we should be consistent.

Reply:
That was the plan for another upcoming PR, but I wanted to keep the changes small here and do the big canvas buffer overhaul separately (see PR description).

Reply:
Sounds good. So `CanvasConfig` will be replaced by `Canvas` eventually?

Reply:
Yes, something like that is the plan :)